controllers/configurationpolicy_controller.go

```diff
@@ -68,6 +70,19 @@ var config *rest.Config
 // NamespaceWatched defines which namespace we can watch for the GRC policies and ignore others
 var NamespaceWatched string
 
+var evalLoopHistogram = prometheus.NewHistogram(
```

Comment:
I've never made an alert off of a histogram - what will that look like? I suspect this will be better than a gauge, but I'm interested to see how it will work (and if there are any non-obvious drawbacks).

Reply:
Note: These are untested queries.

I think a simple alert might be the following. This just takes the average time it took to evaluate all config policies over the last 5 minutes and if it's over 60 seconds, fire an alert.
```promql
(rate(config_policies_evaluation_duration_seconds_sum[5m]) / rate(config_policies_evaluation_duration_seconds_count[5m])) > 60
```

Having it as a histogram rather than a counter allows for checking the percentiles, so something like the following. This checks that at least 99% of the time, it took 30 seconds or less to process all config policies in the past 60 minutes.

```promql
histogram_quantile(.99, sum(rate(config_policies_evaluation_duration_seconds_bucket[60m]))) > 30
```